option optimize_for = LITE_RUNTIME;

message TurntableStartRequest
{
 required uint32 all = 1;
}
